### PR TITLE
[#1] Initialize project using React template

### LINKED ISSUE #1: [Chore] Set up the project using react template
## Why

To set up the project using  [react template ](https://github.com/nimblehq/react-templates) and make sure there's no error

## Acceptance Criteria

- Have a base React application project set up using react template
- The application should be able to run successfully

### PR DESCRIPTION
**Pull request by suphanatjarukulgowit**
- Close #1

## What happened 👀

Create the application using [React template](https://github.com/nimblehq/react-templates) 

## Insight 📝

From discussions about how to PR and use React template properly with @hoangmirs So I re-created the new PR for react template

## Proof Of Work 📹
The application successfully run on [local](http://localhost:3000/)
`npm start`
image
image

### CODE REVIEW

**Comment by bterone** [comment]
Just a question @suphanatjarukulgowit, but why did you delete the `dangerfile.ts`? Did you run into some issues? 👀 

EDIT: Answered on [Slack](https://nimble-co.slack.com/archives/C04JS8UAZNE/p1674098690935529)

**Review by bterone — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ labels: "type : bug"
 
 ## Issue
 
-Describe the issue you are facing. Show us the implementation: screenshots, GIFs, etc.
+Describe the issue you are facing. Show us the implementation: screenshots, GIF, etc.
```
Comment (hoangmirs) [inline .github/ISSUE_TEMPLATE/bug_template.md:10]: Let's follow the latest content of https://github.com/nimblehq/git-template
```suggestion
Describe the issue you are facing. Show us the implementation: screenshots, GIFs, etc.
```

The same for other GitHub templates

File: README.md [hunk 2]
```diff
@@ -1,34 +1,80 @@
-# Git Repository Template
+# React App
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+This project was bootstrapped with [Nimble React template](https://github.com/nimblehq/react-templates).
 
-## Usage
+## Available Scripts
```
Comment (hoangmirs) [inline README.md:5]: I know our template doesn't have the `Prerequisites` section, but we should add it to tell other people which Node version we should use to run this project. The recommended version is the current LTS Node version.

**Review by hoangmirs — APPROVED** [review]
Just 2 small comments

**Pull request merged**